app.js

```diff
@@ -147,10 +155,10 @@ io.on('connection', (socket) => {
 				rooms[i].num_users--;
 				num_users = rooms[i].num_users;
 				for( let j = 0; j < rooms[i].users.length; j++){
-				   if ( rooms[i].users[j] === socket.username) {
-				     rooms[i].users.splice(j, 1);
-				     j--;
-				   }
+					if ( rooms[i].users[j] === socket.username) {
+						rooms[i].users.splice(j, 1);
+						j--;
+					}
```

Comment:
These are just spaces converted into tabs by your text editor. Remove these unnecessary changes.

Reply:
I'll do what I can. Maybe there should be an `.editorconfig` file in this project?

Reply:
I am not in favor of something as strong as  `.editorconfig`, but some guidelines should be there about the codebase in `CONTRIBUTING.md`.